### PR TITLE
CLOUD1156, updated so that both pytest and allure test reports contai…

### PR DESCRIPTION
**Pull request by markg58**
…n same information

Jira issue? CLOUD-1156
Release 6.5.0 backport required? no
6.5.0 doc changes needed? no
7.0.0 doc changes needed? no
Required README updates made? no

### CODE REVIEW

File: cicd/forgeops-tests/tests/conftest.py [hunk 1]
```diff
@@ -1,13 +1,100 @@
...
+    report_path = 'reports'
+    if not os.path.exists(report_path):
+        os.makedirs(report_path)
+
+    set_allure_environment_props(os.path.join(root_dir, report_path, 'allure-files', 'environment.properties'),
+                                 properties)
+
+
+def get_version_info(components):
+    components_list = components.split()
```
Comment (andrewvinall) [inline cicd/forgeops-tests/tests/conftest.py]: components: It may be clearer to pass in a list.

File: cicd/forgeops-tests/tests/conftest.py [hunk 2]
```diff
@@ -1,13 +1,100 @@
...
+        for key, value in version_info.items():
+            config._metadata[key] = value
+            properties[key] = value
+
+    report_path = 'reports'
+    if not os.path.exists(report_path):
+        os.makedirs(report_path)
+
+    set_allure_environment_props(os.path.join(root_dir, report_path, 'allure-files', 'environment.properties'),
+                                 properties)
```
Comment (andrewvinall) [inline cicd/forgeops-tests/tests/conftest.py]: Does the filename need to be parameterised i.e. we could just pass in 'properties'?

File: cicd/forgeops-tests/tests/conftest.py [hunk 3]
```diff
@@ -1,13 +1,100 @@
...
+        if key.startswith("TESTS_"):
+            tests_properties[key] = environment_properties[key]
+    return tests_properties
+
+
+def set_allure_environment_props(filename, properties):
+    # Write properties to environment.properties file
+    with open(filename, 'w') as file:
+        for key, value in properties.items():
+            file.write('%s=%s\n' % (key, value))
```
[the remark this answers is not in the document]
Comment (markg58) [reply cicd/forgeops-tests/tests/conftest.py]: fixed

File: cicd/forgeops-tests/tests/conftest.py [hunk 4]
```diff
@@ -1,13 +1,100 @@
...
+
+    # Get properties that start with TESTS_ from environment.properties
+    tests_properties = {}
+    for key, value in environment_properties.items():
+        if key.startswith("TESTS_"):
+            tests_properties[key] = environment_properties[key]
+    return tests_properties
+
+
+def set_allure_environment_props(filename, properties):
```
[the remark this answers is not in the document]
Comment (markg58) [reply cicd/forgeops-tests/tests/conftest.py]: Added docstring to function.  Added copyright header to the file

File: cicd/forgeops-tests/tests/conftest.py (hunk 1 above)
Comment (markg58) [reply cicd/forgeops-tests/tests/conftest.py]: TESTS_COMPONENTS contains a string currently, so I'm grabbing the string and converting it to a list here.

**Pull request merged**